documentation/configuration.markdown

```diff
@@ -137,6 +137,14 @@ In a default installation, this file doesn't exist. You'll need to create it if
     This setting is used to enable [external SSL termination.](./external_ssl_termination.markdown) If enabled, Puppet Server will ignore any actual certificate presented to the Jetty webserver, and will rely completely on header data to authorize requests. This is very dangerous unless you've secured your network to prevent any untrusted access to Puppet Server.
 
     You can change Puppet's `ssl_client_verify_header` setting to use another header name instead of `X-Client-Verify`; the `ssl_client_header` setting can rename `X-Client-CN`. The `X-Client-Cert` header can't be renamed.
+    
+    Note that this setting only applies to HTTP endpoints served by the "master"
+    service.  The applicable endpoints include those listed in the 
+    "Configuration Management Services", "Informational Services", and
+    "V2 HTTP API" sections of the [Puppet HTTP API](http://docs.puppetlabs.com/references/3.7.latest/developer/file.http_api_index.html)
```

Comment:
@camlow325 It's an artifact of an ancient decision that we later reversed (because frames are a bad tool for displaying anything except ruby method docs). In https://github.com/nfagerlund/puppet/commit/9ce3dbfc27bc34f5b32945eed2ca9f3bc413b5df, I fixed all those on stable, but they're still there in historical versions like 3.8.1.